packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -147,7 +147,7 @@ async function getPluginRelativePath (cfg: LegacyCypressConfigJson, projectRoot:
 // project's node_modules, we don't want to include
 // defineConfig(/***/) in their cypress.config.js,
 // since it won't exist.
-function defineConfigAvailable (projectRoot: string) {
+export function defineConfigAvailable (projectRoot: string) {
```

Comment:
Global mode and weird monorepos! https://cypress-io.atlassian.net/browse/UNIFY-1531

Not that common, but I ran into it during testing. I wanted to update a 9.x project and I was using global mode, so that's how I found it.